ubi8-micro

```diff
@@ -1,12 +1,12 @@
 #
-# Version 1
+# Version 2 for ubi8
 FROM registry.access.redhat.com/ubi8/ubi AS ubi-micro-build
 MAINTAINER Scott McCarty smccarty@redhat.com
 RUN mkdir -p /mnt/rootfs
 RUN yum install --installroot /mnt/rootfs coreutils-single glibc-minimal-langpack --releasever 8 --setopt install_weak_deps=false --nodocs -y
 RUN rm -rf /mnt/rootfs/var/cache/*
-RUN ls -alh /mnt/rootfs/var/cache
+RUN rm -rf /mnt/rootfs/var/lib/rpm
```

Comment:
I'd definitely leave this out to allow introspection of the image contents.

Reply:
That makes sense to me. For my situation I'd rather shave off 3MB than introspect images, but I also rebuild stuff constantly and don't have worry about what's running in the wild. I think it would be cool to have some toggle options and remove this for a deep vacuum of the image.

Reply:
James/Ben,
     Ben knows I hate the rm hacks that we have in our RHEL/UBI 7 images, but sadly we were looking for space savings anywhere we can find it. It's better in RHEL 8 and will get even better in RHEL 9 (fingers crossed, way better). Going to merge as is, just so we have a UBI 7 version.